### DISCUSSION TITLE
Installation freezes when you miss confirmation window

### DISCUSSION THREAD

**Discussion by rafnov** [Droid-ify/client#844] [Ideas]
I can find confirmation sometimes in notifications then, but not always (I don't know what this depends on) and in app window I can do nothing, even cancelling installation does not work. This should be fixed. It is easy to press outside confirmation window when I multitask. Sometimes I have to kill the app and re-run it to go back to installation process - obviously that is a dirty workaround, not a  solution.
Does it encounter on you the same or is it only on me?

**Comment by lemonadeforlife** [comment]
I faced the same issue that's why I use Legacy Installer instead.

**Comment by J-HS-L** [comment]
This should lodged as an "issue"
After 1st checking its not already lodged
follow the guidelines for submissions there, ofc.
